website/docs/docs/dbt-cloud/dbt-cloud-enterprise/setting-up-sso-with-google-gsuite.md

```diff
@@ -134,9 +141,6 @@ If OAuth verification does not complete successfully, double check that:
  - The Client ID and Client Secret provided match the values generated in the
    GCP Credentials page
  - An Authorized Domain was provided in the OAuth Consent Screen configuration
-
-### Missing groups
```

Comment:
@matt-winkler did you mean to remove this header?

Reply:
@carlyfk Yes, because there is no additional text underneath the header to explain the topic, so it didn't seem that useful. Is preference to keep it in there?

Reply:
I'm cool with removing the header, but FWIW markdown headers become linkable, which might be convenient if you'd like to link someone to a specific section of the documentation. Call it how you see it on this one.

Reply:
Cool thanks for reviewing. Makes sense on headers being linkable. Thinking I'll continue testing and add back with more specifics once there is more information to include.